docs/how-to-write-a-contract/built-ins.md

```diff
@@ -195,6 +195,18 @@ const demo = new Demo()
 const size = demo.scriptSize
 ```
 
+### `loadArtifact`
+
+Function `static loadArtifact(artifact: MergedArtifact)` loads the contract artifact file in order to rebuild a contract instance, it's usually called on the front end.
```

Comment:
, it's usually called on the front end.
->
. It's usually called at the front end.